MarkBind/markbind
# Fix filename capitalisation for syntax documentation

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [X] Documentation update






**What is the rationale for this request?**
The current user documentation has broken includes on Netlify due to [case sensitive filenames](https://www.netlify.com/docs/build-gotchas/#case-sensitivity):

```
  __  __                  _      ____    _               _
 |  \/  |   __ _   _ __  | | __ | __ )  (_)  _ __     __| |
 | |\/| |  / _` | | '__| | |/ / |  _ \  | | | '_ \   / _` |
 | |  | | | (_| | | |    |   <  | |_) | | | | | | | | (_| |
 |_|  |_|  \__,_| |_|    |_|\_\ |____/  |_| |_| |_|  \__,_|
 v1.17.3
info: Website generation started at 08:54:12
info: Building assets...
info: Assets built
info: Generating pages...
error: Error: No such file: /opt/build/repo/docs/userGuide/syntax/navBars.mbdf
Missing reference in /opt/build/repo/docs/userGuide/fullSyntaxReference.md
error: Error: No such file: /opt/build/repo/docs/userGuide/syntax/searchBars.mbdf
Missing reference in /opt/build/repo/docs/userGuide/fullSyntaxReference.md
error: Error: No such file: /opt/build/repo/docs/userGuide/syntax/textStyles.mbdf
Missing reference in /opt/build/repo/docs/userGuide/markBindSyntaxOverview.md
error: Error: No such file: /opt/build/repo/docs/userGuide/syntax/textStyles.mbdf
Missing reference in /opt/build/repo/docs/userGuide/markBindSyntaxOverview.md
error: Error: No such file: /opt/build/repo/docs/userGuide/syntax/searchBars.mbdf
Missing reference in /opt/build/repo/docs/userGuide/readerFacingFeatures.md
error: Error: No such file: /opt/build/repo/docs/userGuide/syntax/searchBars.mbdf
Missing reference in /opt/build/repo/docs/userGuide/readerFacingFeatures.md
error: Error: No such file: /opt/build/repo/docs/userGuide/syntax/navBars.mbdf
Missing reference in /opt/build/repo/docs/userGuide/readerFacingFeatures.md
error: Error: No such file: /opt/build/repo/docs/userGuide/syntax/navBars.mbdf
Missing reference in /opt/build/repo/docs/userGuide/readerFacingFeatures.md
error: Error: No such file: /opt/build/repo/docs/userGuide/syntax/searchBars.mbdf
Missing reference in /opt/build/repo/docs/userGuide/syntaxCheatSheet.md
error: Error: No such file: /opt/build/repo/docs/userGuide/syntax/searchBars.mbdf
Missing reference in /opt/build/repo/docs/userGuide/syntaxCheatSheet.md
error: Error: No such file: /opt/build/repo/docs/userGuide/syntax/navBars.mbdf
Missing reference in /opt/build/repo/docs/userGuide/syntaxCheatSheet.md
error: Error: No such file: /opt/build/repo/docs/userGuide/syntax/navBars.mbdf
Missing reference in /opt/build/repo/docs/userGuide/syntaxCheatSheet.md
error: Error: No such file: /opt/build/repo/docs/userGuide/syntax/navBars.mbdf
Missing reference in /opt/build/repo/docs/userGuide/usingComponents.md
```

**What changes did you make? (Give an overview)**
I renamed these files to have camelCase as expected:
- `docs/userGuide/syntax/navbars.mbdf` → `docs/userGuide/syntax/navBars.mbdf`
- `docs/userGuide/syntax/searchbars.mbdf` → `docs/userGuide/syntax/searchBars.mbdf`
- `docs/userGuide/syntax/textstyles.mbdf` → `docs/userGuide/syntax/textStyles.mbdf`

**Testing instructions:**
1. The Netlify deploy preview for this PR should build without errors, and the relevant sections should be included as normal.
